packages/core/src/enqueue_links/enqueue_links.ts

```diff
@@ -55,6 +55,18 @@ export interface EnqueueLinksOptions extends RequestQueueOperationOptions {
      */
     globs?: GlobInput[];
 
+    /**
+     * An array of glob pattern strings or plain objects
+     * containing glob pattern strings matching URLs that will **never** be enqueued.
+     *
+     * The plain objects must include at least the `glob` property, which holds the glob pattern string.
+     * All remaining keys will be used as request options for the corresponding enqueued {@apilink Request} objects.
+     *
+     * The matching is always case-insensitive.
+     * If you need case-sensitive matching, use `regexps` property directly.
+     */
+    blacklist?: GlobInput[];
```

Comment:
Named the option `blacklist`. Technically speaking, I think my implementation is ready to accept regex blacklists similar to `glob`/`regexps` though in this case it is solely globs. Let me know if we should adjust or rename this.

Reply:
maybe `exclude` would be a better name? re the regexps, i guess its fine to support globs only, maybe we could allow the `regexp` key in the object too? i dont think we need two separate options for blacklisting globs and regexps separately

thoughts @barjin @vladfrangu?

Reply:
Pushed a commit to access globs or regex :)